modules/ztp-creating-the-site-secrets.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="ztp-creating-the-site-secrets_{context}"]
+= Creating the site secrets
+
+Add the required secrets for the site to the hub cluster. These resources must be in a namespace with a name that matches the cluster name.
+
+.Procedure
+
+. Create a secret for authenticating to the site BMC. Ensure the secret name matches the name used in the SiteConfig.
+In this example, the secret name is `test-sno-bmh-secret`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: test-sno-bmh-secret
+  namespace: test-sno
+data:
+  password: dGVtcA==
+  username: cm9vdA==
+type: Opaque
+----
+
+. Create the pull secret for the site. The pull secret must contain all credentials necessary for installing OpenShift and all day-2 operators. In this example, the secret name is `assisted-deployment-pull-secret`:
```

Comment:
s/OpenShift/{product-title}
s/day-2 operators/add-on Operators

Reply:
Our product title is OpenShift Container Platform.
2nd changes made.